migrations/2_deploy.js

```diff
@@ -57,6 +54,11 @@ module.exports = async function(deployer, network, accounts) {
   const zTokenInstance = await ZDai.deployed();
   await zTokenInstance.addMinter(LendingPool.address, { from: deployerAccount });
```

Comment:
I think as you removed the LendingPool.zTokenMinted function, this "addMinter" role is not needed now.

Reply:
actually `burn` is marked as `onlyMinter` so we still need the minter role